packages/affirm-integration/src/affirm-payment-strategy.ts

```diff
@@ -235,34 +224,34 @@ export default class AffirmPaymentStrategy implements PaymentStrategy {
         return shippingInformation;
     }
 
-    private _getItems(amountTransformer: AmountTransformer, order: Order): AffirmItem[] {
+    private getItems(amountTransformer: AmountTransformer, order: Order): AffirmItem[] {
```

Comment:
In case that all of the items has different type, we will need to add multiple type guards, and can't simplify code there. So better to keep as it is. Thanks Sebastian for check!